packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -152,7 +153,7 @@ const GlobalHeader = ({
     }
   `);
 
-  const hideLogoText = useMedia({ maxWidth: '350px' });
+  const hideLogoText = useMedia({ maxWidth: '460px' });
```

Comment:
are we trying to keep this in sync with the `LOGO_TEXT_BREAKPOINT` constant? if so, would it make sense to use that here?

Reply:
hohoho it sure would, lemme fix that